app/src/main/java/org/oppia/app/story/StoryActivity.kt

```diff
@@ -15,23 +15,39 @@ class StoryActivity : InjectableAppCompatActivity(), RouteToExplorationListener
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     activityComponent.inject(this)
-    val storyId: String = checkNotNull(intent.getStringExtra(STORY_ACTIVITY_INTENT_EXTRA)) {
+    val internalProfileId = intent.getIntExtra(STORY_ACTIVITY_INTENT_EXTRA_INTERNAL_PROFILE_ID, -1)
```

Comment:
Please follow one format in naming intent keys.

Reply:
I agree with your point here. For this I have filed an issue: https://github.com/oppia/oppia-android/issues/747

Because this is something which is not a very high priority task. Also, once we finalise what convention we want to follow, it would can be `good-first-issue`.